### PR TITLE
nodejs-ar: init Arabic localization

### PR DESCRIPTION
**Pull request by ouadie-lahdioui**
Hello,

The aim of this PR is to initialize an Arabic locale for www.nodejs.org website to help arabic developers get involved.

### CODE REVIEW

**Comment by eljoujat** [comment]
+1

**Comment by jseijas** [comment]
Hi! Anyone able to review this PR? A +1 from a native speaker is needed before merging. 

And very good job and many thanks!

**Comment by sotayamashita** [comment]
@ouadie-lahdioui I would appreciate if you could know any other person who can review.

File: locale/ar/site.json [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+      "text": "Conduct"
+    }
+  },
+  "security": {
+    "link": "security",
+    "text": "أمن"
+  },
+  "blog": {
+    "link": "blog",
+    "text": "أخبار"
```
Comment (AbdelghaniAzri) [inline locale/ar/site.json]: blog المرجو استعمال مدونة كترجمة ل

Comment (ouadie-lahdioui) [reply locale/ar/site.json]: Done, thanks.

**Review by AbdelghaniAzri — APPROVED** [review]
Ok good job.

**Comment by fhemberger** [comment]
There are still some translations for labels missing in `site.json`, can you translate them as well or do you need help?

**Comment by AbdelghaniAzri** [comment]
Ok.

**Comment by fhemberger** [comment]
By the way: Arabic is our first RTL translation. Is the overall layout looking okay for you? Do we need to update the CSS?

**Comment by zmazouzi** [comment]
+1 from Arabic native speaker... thank you

File: locale/ar/site.json [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      "link": "get-involved/contribute",
+      "text": "ساهم"
+    },
+    "development": {
+      "link": "get-involved/development",
+      "text": "تطوير"
+    },
+    "conduct": {
+      "link": "https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#code-of-conduct",
+      "text": "سلوك"
```
Comment (mouadennaciri) [inline locale/ar/site.json]: القواعد السلوكية ؟

**Review by mouadennaciri — COMMENTED** [review]
Good job

**Comment by HACHIMIam** [comment]
+1

**Comment by clakech** [comment]
Still need Arabic speaker reviewer?

**Comment by ghost** [comment]
@clakech it would be great if you could review it as well

File: locale/ar/site.json (hunk 2 above)
Comment (ouadie-lahdioui) [reply locale/ar/site.json]: Done, thanks.

**Review by mouadennaciri — APPROVED** [review]
LGTM 👍

**Comment by mohamed-taman** [comment]
+1 for Arabic translation

**Comment by fhemberger** [comment]
Merged! Thank you all for your work! :tada:

**Comment by ouadie-lahdioui** [comment]
🎉 🎉 Yataa 🎉 🎉

@fhemberger i'll open a new issue for some css updates.

special thanks to all who reviewed

**Pull request merged**